### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by 0xLeif**

### CODE REVIEW

File: olGO/API.swift [hunk 1]
```diff
@@ -79,8 +79,8 @@ class API {
         API()
     }()
     // Configuations
-    let path = "http://localhost:8080"
-    //    let path: String = "https://olapi-develop.vapor.cloud"
+    let path = "http://localhost:8080/api"
+    //    let path: String = "https://olapi-develop.vapor.cloud/api"
```
Comment (ambid17) [inline olGO/API.swift:83]: ticket added to make this modular

File: olGO/Views/Auth/RegisterView.swift [hunk 2]
```diff
@@ -76,6 +76,14 @@ class RegisterView: UIView {
                         Navigate.shared.go(UIViewController {
                             LoginView()
                         }, style: .push)
+                        
+                        DispatchQueue.main.async {
+                            Navigate.shared.alert(title: "You have been Logged out...", message: "Whoops...") {_ in }
+                            DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```
Comment (ambid17) [inline olGO/Views/Auth/RegisterView.swift:82]: magic "+ 1"?

**Review by ambid17 — APPROVED** [review]
one change requested, merge as is or fix, its up to you

File: olGO/Views/Auth/RegisterView.swift (hunk 2 above)
Comment (0xLeif) [reply olGO/Views/Auth/RegisterView.swift:82]: .now() is normal and then you add how many seconds you want. so 1 second

**Pull request merged**